package.json

```
@@ -40,85 +40,43 @@
     }
   ],
   "exports": {
-    "types": "./dist/index.d.ts",
-    "import": "./dist/esm/romcal.js",
-    "require": "./dist/cjs/romcal.js"
+    "types": "./rites/roman1969/dist/index.d.ts",
+    "import": "./rites/roman1969/dist/esm/romcal.js",
+    "require": "./rites/roman1969/dist/cjs/romcal.js"
   },
-  "main": "dist/cjs/romcal.js",
-  "module": "dist/esm/romcal.js",
-  "typings": "dist/index.d.ts",
+  "main": "rites/roman1969/dist/cjs/romcal.js",
+  "module": "rites/roman1969/dist/esm/romcal.js",
+  "typings": "rites/roman1969/dist/index.d.ts",
   "scripts": {
-    "build": "node -r ts-node/register scripts/build.ts",
-    "clean": "rimraf coverage dist tmp",
-    "data:checks": "node -r ts-node/register scripts/data-checks.ts",
-    "doc": "node -r ts-node/register scripts/bundle-doc.ts",
-    "docs:check-links": "node scripts/docs/check-broken-links.mjs ./docs",
-    "docs:sort-glossary": "node scripts/docs/sort-glossary.mjs ./docs",
-    "lint": "eslint .",
-    "lint:fix": "npm run lint -- --fix",
+    "build": "npm run build -w=@internal/easter && npm run build -w=@internal/rite-roman1969",
+    "clean": "rimraf coverage dist tmp && npm run clean -w=@internal/easter && npm run clean -w=@internal/rite-roman1969",
+    "data:checks": "npm run data:checks -w=@internal/rite-roman1969",
+    "doc": "npm run doc -w=@internal/rite-roman1969",
+    "docs:check-links": "npm run docs:check-links -w=@internal/rite-roman1969",
+    "docs:sort-glossary": "npm run docs:sort-glossary -w=@internal/rite-roman1969",
+    "lint": "npm run lint -w=@internal/easter && npm run lint -w=@internal/rite-roman1969",
+    "lint:fix": "npm run lint -w=@internal/easter && npm run lint:fix -w=@internal/rite-roman1969",
```

as above here (for running multiple workspaces at once), but it looks like the first command lost the fix flag, so we should have...
```suggestion
    "lint:fix": "npm run lint -w=@internal/easter -w=@internal/rite-roman1969  -- --fix",
```